.github/workflows/pr-check.yaml

```diff
@@ -120,16 +120,20 @@ jobs:
       - name: Build Podman Desktop for E2E tests
         working-directory: ./podman-desktop
         run: |
-          pnpm install
+          pnpm install --frozen-lockfile
           pnpm test:e2e:build
 
       - name: Ensure getting current HEAD version of the test framework
         working-directory: ./podman-desktop-redhat-account-ext
-        run: pnpm install -D @podman-desktop/tests-playwright@next 
+        run: |
+          # workaround for https://github.com/containers/podman-desktop-extension-bootc/issues/712
+          version=$(npm view @podman-desktop/tests-playwright@next version)
+          echo "Version of @podman-desktop/tests-playwright to be used: $version"
+          jq --arg version "$version" '.devDependencies."@podman-desktop/tests-playwright" = $version' package.json > package.json_tmp && mv package.json_tmp package.json
 
       - name: Execute pnpm in SSO Extension
         working-directory: ./podman-desktop-redhat-account-ext
-        run: pnpm install
+        run: pnpm install --no-frozen-lockfile
```

Comment:
It's added automatically in ci env so no need

Reply:
I think that default is to do not generate lock-file in CI env. So we need to turn it of since we need to update the lock file, right?
```
--[no-]frozen-lockfile            Don't generate a lockfile and fail if an update is needed. This setting is on by default in
                                        CI environments, so use --no-frozen-lockfile if you need to disable it for some reason
```